### ISSUE TITLE
Adding an option to use a custom loss function 

### ISSUE THREAD

**Issue by GalOf-s** [ThilinaRajapakse/simpletransformers#751]
I'm using your library to solve multi-label document classification, for the past six months, and it's great! 

A recent ([paper](https://arxiv.org/abs/2009.14119)) proposed a new asymmetric loss function for multi-label classification problems which i would like to try out on my problem, but unfortunately as far as i could tell, it's not currently possible, since the loss function is hard coded at [custom_models/models.py ](https://github.com/ThilinaRajapakse/simpletransformers/blob/master/simpletransformers/custom_models/models.py) (at least for BERTY which I'm using).

I would be glad if it was possible to simply configure the loss function as another field in `args`, with cross-entropy being the obvious default.

I'm also linking a [pytorch implementation ](https://github.com/Alibaba-MIIL/ASL/blob/main/src/loss_functions/losses.py) of the new loss I want to test out, to give a sense of what type of options I would like to use that argument.

Thanks a lot!

**Comment by ThilinaRajapakse** [comment]
This would be useful. I've added it as a feature request and I'll try to get around to it quickly, but I am kinda busy these days so there might be delays.
